variants/heltec_wireless_tracker_V1_1/platformio.ini

```diff
@@ -0,0 +1,14 @@
+[env:heltec-wireless-tracker]
+extends = esp32s3_base
+board = heltec_wireless_tracker
+upload_protocol = esp-builtin
+
+build_flags = 
+  ${esp32s3_base.build_flags} -I variants/heltec_wireless_tracker_V1_1
+  -D HELTEC_TRACKER_V1_1
```

Comment:
I believe this needs to be changed to have the correct HW_MODEL selection again in architecture.h

Reply:
should i change heltec_wireless_tracker_V1_1 to heltec_wireless_tracker and the relative folder name?

Reply:
That means the main_matrix.yml would need changed back as well if we've reverted Gabriel's protobuf commit right?

Reply:
Variants adds only the 1.0 version, should I just change the folder name of the 1.1?

Reply:
Yes, I think folder name, variant, and the -D HELTEC_TRACKER_V1_1

Reply:
So we should completely drop the support to V1.0 PCB and leave only the names as before?

Reply:
I realize I wasn't very clear. 
The 1.0 variant is fine, but we want the platformio assets for the 1.1 variant to remain (heltec_wireless_tracker) for backwards compatibility of the firmware flashing API collecting those assets. v1.1 is also the most common version folks have. Splitting this target into two discrete ones will surely involve some support cleanup effort, but this will minimize it a bit.